packages/cfd/src/Constants/dynamic-leverage-content/dynamic-leverage-content.ts

```diff
@@ -0,0 +1,111 @@
+import { localize } from '@deriv/translations';
+
+export const dynamicLeverages = [
```

Comment:
```suggestion
export const dynamic_leverages = [
```
naming convention

Reply:
@arshad-rao-deriv you can check out the README file here
https://github.com/binary-com/deriv-app/tree/master/docs/JavaScript#naming-conventions

Reply:
Updated.